src/cljc/athens/util.cljc

```diff
@@ -10,3 +10,14 @@
   []
   (-> (js/Date.) .getTime))
 
+
+(defn scrollIfNeeded
```

Comment:
Can use kebab-case now. Not necessary for JS props, but makes sense for the function name